chapters/fundamentals.adoc

```diff
@@ -877,12 +877,11 @@ Specification_, can: be `NULL`.
 [[fundamentals-validusage-enums]]
 ==== Valid Usage for Enumerated Types
 
-Any parameter of an enumerated type must: be a valid enumerant for that
-type.
-Use of an enumerant is valid if the following conditions are true:
+An enumerant value is valid for a particular use if the following conditions
+are true:
```

Comment:
maybe "all the following conditions".

Reply:
We're inconsistent about whether we use "all". I agree it wouldn't hurt!